apps/sumo-aws-apps/security-hub/securityhub.template.yaml

```diff
@@ -40,7 +40,6 @@ Metadata:
           default: "AWS Quick Start configuration"
         Parameters:
           - QSS3BucketName
-          - QSS3BucketRegion
```

Comment:
Why are we removing bucket region ?

Reply:
because the  Parameter QSS3BucketRegion not used.